enterprise/reporting/flusher/flusher.go

```diff
@@ -107,6 +106,15 @@ func NewFlusher(db *sql.DB, log logger.Logger, stats stats.Stats, conf *config.C
 
 	f.initCommonTags()
 	f.initStats(f.commonTags)
+	if _, err := db.Exec(
+		fmt.Sprintf("vacuum full analyze %s", pq.QuoteIdentifier(table)),
+	); err != nil {
+		log.Errorn(
+			fmt.Sprintf(`Error full vacuuming %s table`, table),
```

Comment:
The point of using a non-sugared logger is to avoid reflection (which happens with `fmt.Sprintf`. Also, please use the observability kit helpers whenever possible e.g. `obskit.Error`.

```go
log.Errorn("Cannot do full vacuum analyze", logger.NewStringField("table", table), obskit.Error(err))
```